lib/kino_explorer/data_transform_cell.ex

```diff
@@ -280,6 +280,24 @@ defmodule KinoExplorer.DataTransformCell do
     {:noreply, ctx}
   end
 
+  def handle_event("move_inner_value", fields, ctx) do
+    {field, idx, remove, add} =
+      {fields["field"], fields["idx"], fields["removedIndex"], fields["addedIndex"]}
+
+    {value, values} =
+      get_in(ctx.assigns.operations, [Access.at(idx), field]) |> List.pop_at(remove)
+
+    updated_value = List.insert_at(values, add, value)
+
+    updated_operations =
+      put_in(ctx.assigns.operations, [Access.at(idx), field], updated_value)
+      |> update_data_options(ctx)
```

Comment:
We can merge `get_in + put_in -> update_in` :D

```suggestion
    updated_operations =
      ctx.assigns.operations
      |> update_in([Access.at(idx), field], fn values ->
        {value, values} = List.pop_at(values, remove)
        List.insert_at(values, add, value)
      end)
      |> update_data_options(ctx)
```